twisted/internet/iocpreactor/tcp.py

```diff
@@ -59,7 +60,13 @@ def getHandle(self):
 
     def dataReceived(self, rbuffer):
         # XXX: some day, we'll have protocols that can handle raw buffers
-        self.protocol.dataReceived(str(rbuffer))
+        if not isinstance(rbuffer, bytes):
+            if _PY3:
```

Comment:
Anywhere we use `_PY3` it's a potential future maintenance problem.  We should strive to always use the relevant high-level API from twisted.python.compat.  In this case though I don't know what type the `rbuffer` argument might be, so it's hard to be sure what exactly this should be.  I'm like 90% sure that `"utf-8"` is the wrong thing to say below, but without being surer of what this is I don't know what else to suggest.

To address this, as per [policy](https://twistedmatrix.com/trac/wiki/ReviewProcess#Authors:Thingsyourbranchorpatchmustcontain), all new or changed methods must have 100% docstring coverage, so if you could add a docstring here that says what types `rbuffer` is allowed to be.